claim/claim.go

```diff
@@ -0,0 +1,95 @@
+package claim
+
+import (
+	"fmt"
+	"math/rand"
+	"regexp"
+	"time"
+
+	"github.com/oklog/ulid"
+
+	"github.com/deislabs/cnab-go/bundle"
+)
+
+// Status constants define the CNAB status fields on a Result.
+const (
+	StatusSuccess  = "success"
+	StatusFailure  = "failure"
+	StatusUnderway = "underway"
+	StatusUnknown  = "unknown"
+)
+
+// Action constants define the CNAB action to be taken
+const (
```

Comment:
Would it be worth introducing an `Action` type (based on `string`) to avoid actions being accidentally intermixed with raw strings? Similarly for `Status` above.